woocommerce/woocommerce-gutenberg-products-block
# Button: Update deprecated props

## What

Fixes #11855

## Why

This PR updates the deprecated `isPrimary`, `isSecondary`, `isTertiary` and `isLink` props to the `variant` prop.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page
2. Add test blocks from the file: [testblocks](HTTP://cloudup.com/cTsFpZodJye)
3. Make sure that all the updated buttons work/appear correctly.


## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.

PR description updated by @danieldudzic
